src/Nethermind/Nethermind.Network/Config/INetworkConfig.cs

```diff
@@ -100,4 +100,10 @@ public interface INetworkConfig : IConfig
 
     [ConfigItem(DefaultValue = "false", HiddenFromDocs = true, Description = "[TECHNICAL] Disable feeding ENR DNS records to discv4 table")]
     bool DisableDiscV4DnsFeeder { get; set; }
+
+    [ConfigItem(DefaultValue = "0", HiddenFromDocs = true, Description = $"[TECHNICAL] Introduce throttling to all messages being sent/received after exceeding a byte threshold in a given time span (see '{nameof(ThrottlingDelayMs)}')")]
+    int ThrottlingByteLimit { get; set; }
+
+    [ConfigItem(DefaultValue = "0", HiddenFromDocs = true, Description = $"[TECHNICAL] Introduce throttling to all messages being sent/received after exceeding a byte threshold in a given time span (see '{nameof(ThrottlingByteLimit)}')")]
+    int ThrottlingDelayMs { get; set; }
```

Comment:
New (technical) config available here: by default, no throttling is used.

Reply:
Let's drop `Ms` from `ThrottlingDelayMs`.
Also, please elaborate on these parameters. Their descriptions should be different. Even though they are excluded from the docs now, we need a better explanation as they are in the `ConfigItem` attributes. I'll improve those if you give me more context.